docs/sources/developers/plugins/metadata.md

```diff
@@ -16,7 +16,7 @@ The plugin.json file is required for all plugins. When Grafana starts, it scans
 
 | Property        | Type                    | Required | Description                                                                                                                                                                                                                                                                                                                                                                                             |
 |-----------------|-------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `dependencies`  | [object](#dependencies) | **Yes**  | Plugin dependencies.                                                                                                                                                                                                                                                                                                                                                                                    |
+| `dependencies`  | [object](#dependencies) | **Yes**  | Dependencies needed by the plugin.                                                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I'm super torn on this property -- I think it was added aspirationally... it has no real effect.  It will list the dependencies on the grafana.com page, but that is it.

Reply:
That's my understanding as well. We should add it to the description at least.